### PR TITLE
Refactor cloud connection network attach resource and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go [hunk 1]
```diff
@@ -13,13 +13,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
 	st "github.com/IBM-Cloud/power-go-client/clients/instance"
-	"github.com/IBM-Cloud/power-go-client/helpers"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
 )
 
 const (
-	PICloudConnectionNetworkId = "pi_network_id"
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go:22]: According to the documentation:
https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/pi_cloud_connection_network_attach

cloud instance, cloud connection and cloud connection network id are all arguments.
Please relabel as Arg_.

Also this definition to ibm_pi_constants.go.

File: ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go [hunk 2]
```diff
@@ -36,19 +35,19 @@ func ResourceIBMPICloudConnectionNetworkAttach() *schema.Resource {
...
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
 				Description: "PI cloud instance ID",
 			},
-			helpers.PICloudConnectionId: {
+			Attr_CloudConnectionID: {
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go:44]: helpers.PICloudConnectionId -> pi_cloud_connection_id
Attr_CloudConnectionID -> cloud_connection_id 
we can't redefine the variables in the schema.
Create a new variable called Arg_CloudConnectionID that has pi_cloud_connection_id as the value and put in the constants file.

File: ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go [hunk 3]
```diff
@@ -36,19 +35,19 @@ func ResourceIBMPICloudConnectionNetworkAttach() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
 				Description: "PI cloud instance ID",
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go:42]: Sort these fields (type, requried, forcenew description) alphabetically, and match their description to what's in the documentation.

File: ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go [hunk 4]
```diff
@@ -13,13 +13,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
 	st "github.com/IBM-Cloud/power-go-client/clients/instance"
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go:15]: Remove the st shortname and merge the github.com import blocks.

File: website/docs/r/pi_network_port_attach.html.markdown [hunk 5]
```diff
@@ -8,6 +8,7 @@ description: |-
 ---
 
 # ibm_pi_network_port_attach
```
Comment (ismirlia) [inline website/docs/r/pi_network_port_attach.html.markdown:10]: This is the wrong documentation file for this resource. The file you are looking for is pi_cloud_connection_network_attach.html.markdown.

**Comment by Diptipowervs** [comment]
close PR due to some conflicts in file

**Pull request closed without merging**